book/src/SDK/platform/mongodb/migrations.md

```diff
@@ -0,0 +1,55 @@
+## Migrations
+
+Sometimes, it is necessary to refactor a data model after it has already been release to 
+production. The `migrations` module provides a mechanism to both `apply` and `rollback` a migration.
```

Comment:
> provides a mechanism 

"provides mechanisms"?  Or are the apply and rollback part of the same mechanism?

Reply:
If we are talking about multiple mechanisms, then the below text will have to be modified as well.

Reply:
Updated in next push